src/main/java/org/resitrack/controller/PersonController.java

```diff
@@ -4,42 +4,85 @@
 import org.resitrack.enums.Gender;
 import org.resitrack.util.CommonUtil;
 
-import static org.resitrack.util.CommonUtil.INITIAL_ID_VALUE;
-
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Scanner;
 import java.util.concurrent.atomic.AtomicInteger;
 
+import static org.resitrack.util.CommonUtil.INITIAL_ID_VALUE;
+
 
 public class PersonController {
     private static final AtomicInteger count = new AtomicInteger(INITIAL_ID_VALUE);
-    private final List<Person> people;
-    private final Scanner scanner;
+    private List<Person> people;
+    Scanner scanner = new Scanner(System.in);
+
+    public PersonController() {
 
-    public PersonController(List<Person> people, Scanner scanner) {
+    }
+
+    public void setPeople(List<Person> people) {
         this.people = people;
-        this.scanner = scanner;
     }
 
-    public void addNewPerson() {
-        System.out.println("Adding new person");
+    /**
+     * This function is used to add new one person
+     * @return true if add process success and vice versa
+     */
+
+    public boolean addNewPerson() {
         System.out.print("Enter person name: ");
         String name = scanner.nextLine();
 
         Gender gender = getGender();
         Person person = createPerson(name, gender);
 
-        if (!addPerson(person)) {
-            System.out.println("Failed to add a new person");
-        } else {
-            System.out.println("Added person: " + person);
+        return addPerson(person);
+    }
+
+    /**
+     * This function is used to show all person be added
+     */
+    public void showAll() {
+        System.out.format("| %-4s | %-20s | %-10s | %-6s | %-12s |\n", "ID", "Name", "ID Number", "Sex", "Date of Birth");
+        System.out.println("---------------------------------------------------------------------");
+        for (Person person : people) {
+            System.out.format("| %-4s | %-20s | %-10s | %-6s | %-12s  |\n",
+                    person.getId(),
+                    person.getName(),
+                    person.getIdNumber(),
+                    person.getGender(),
+                    person.getDob().toString()
+            );
         }
+        System.out.println("---------------------------------------------------------------------");
     }
 
+    /**
+     * This function is used to delete one person
+     *
+     * @param id
```

Comment:
please explain where is this param come from, what is value of it

Reply:
Đã update